utils/remote.c

```diff
@@ -645,6 +647,22 @@ static void addDays(tr_variant* args, tr_quark const key, char const* arg)
     }
 }
 
+static void addLabels(tr_variant* args, const tr_quark key, const char* arg)
+{
+    tr_variant* labels = tr_variantDictAddList(args, key, 100);
```

Comment:
preallocating 100 label slots seems like a lot?

Reply:
I believe original author copied this from function below but 100 files is definitely more likely than 100 labels. I changed this to 10, does this work better?